bears/js/JSONFormatBear.py

```diff
@@ -1,5 +1,7 @@
 import json
+from os import stat
```

Comment:
As @Makman2 suggested, use `len`, `stat` no required. Also, if the file has say a newline, or any other whitespace character, and is not empty, then your code will give the same error since the file would not be empty in size but actually is a `null` JSON.